### ISSUE TITLE
Create new email newsletter sign-up page

### ISSUE THREAD

**Issue by smarziano** [smartchicago/chicago-early-learning#732]
Before the application launches this year, our partners want to encourage parents to sign up for the Chicago Early Learning Keep in Touch email. Currently, when parents click on the "Early Learning Email Newsletter" link they are led to this URL: http://chicagoearlylearning.us5.list-manage.com/subscribe?u=085247dea37361c266002462c&id=5161f6a47c

We want to create an easier URL to share/remember like: chicagoearlylearning.org/email

Therefore, let's create a new page and embed the Mailchimp sign-up form. 

Here is the embed code:

`
<link href="//cdn-images.mailchimp.com/embedcode/classic-10_7.css" rel="stylesheet" type="text/css">
<style type="text/css">
	#mc_embed_signup{background:#fff; clear:left; font:14px Helvetica,Arial,sans-serif; }
	/* Add your own MailChimp form style overrides in your site stylesheet or in this style block.
	   We recommend moving this block and the preceding CSS link to the HEAD of your HTML file. */
</style>
<div id="mc_embed_signup">
<form action="//chicagoearlylearning.us5.list-manage.com/subscribe/post?u=085247dea37361c266002462c&amp;id=5161f6a47c" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
    <div id="mc_embed_signup_scroll">
	<h2>Subscribe to our mailing list</h2>
<div class="indicates-required"><span class="asterisk">*</span> indicates required</div>
<div class="mc-field-group">
	<label for="mce-EMAIL">Email Address  <span class="asterisk">*</span>
</label>
	<input type="email" value="" name="EMAIL" class="required email" id="mce-EMAIL">
</div>
<div class="mc-field-group">
	<label for="mce-FNAME">First Name  <span class="asterisk">*</span>
</label>
	<input type="text" value="" name="FNAME" class="required" id="mce-FNAME">
</div>
<div class="mc-field-group">
	<label for="mce-LNAME">Last Name  <span class="asterisk">*</span>
</label>
	<input type="text" value="" name="LNAME" class="required" id="mce-LNAME">
</div>
<div class="mc-field-group">
	<label for="mce-MMERGE4">Zip Code  <span class="asterisk">*</span>
</label>
	<input type="text" value="" name="MMERGE4" class="required" id="mce-MMERGE4">
</div>
<div class="mc-field-group size1of2">
	<label for="mce-MMERGE5">Text Me! Here's my cell phone number </label>
    <div class="phonefield phonefield-us">
        (<span class="phonearea"><input class="phonepart " pattern="[0-9]*" id="mce-MMERGE5-area" name="MMERGE5[area]" maxlength="3" size="3" value="" type="text"></span>)
        <span class="phonedetail1"><input class="phonepart " pattern="[0-9]*" id="mce-MMERGE5-detail1" name="MMERGE5[detail1]" maxlength="3" size="3" value="" type="text"></span> - 
        <span class="phonedetail2"><input class="phonepart " pattern="[0-9]*" id="mce-MMERGE5-detail2" name="MMERGE5[detail2]" maxlength="4" size="4" value="" type="text"></span>
		<span class="small-meta nowrap">(###) ###-####</span>
    </div>
</div><div class="mc-field-group">
	<label for="mce-MMERGE3">Source </label>
	<input type="text" value="" name="MMERGE3" class="" id="mce-MMERGE3">
</div>
	<div id="mce-responses" class="clear">
		<div class="response" id="mce-error-response" style="display:none"></div>
		<div class="response" id="mce-success-response" style="display:none"></div>
	</div>    
    <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_085247dea37361c266002462c_5161f6a47c" tabindex="-1" value=""></div>
    <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button"></div>
    </div>
</form>
</div>

`

**Comment by smarziano** [comment]
This new page was created via https://github.com/smartchicago/chicago-early-learning/pull/735.

Assigning this to myself to see what text (if any) should be at the top of the page. We might redirect parents who click on apply links to sign-up and receive notifications when the application is live. This is a step before we can complete #731.

**Comment by smarziano** [comment]
Here are a few tasks to complete on this page:

**@theandrewbriggs:**
- [ ] In the browser tab it currently says "About" but should say "Connect"
<img width="487" alt="screen shot 2017-05-05 at 1 25 33 pm" src="https://cloud.githubusercontent.com/assets/5550969/25759007/5b397c7a-3196-11e7-9ac3-ea094c08fd0a.png">

- [ ] There should be a sentence at the top of the page: "Stay up to date on Chicago Early Learning! Join our email and text lists."

**Me:**
- [ ] Add a preferred communication question checkbox on the email form. "Check off how you'd like to receive communications."
- [ ] Send new embed code
- [ ] Update email form to include English and Spanish language. 
- [ ] Get Spanish translation of the top "Stay up to date..." sentence

**Comment by smarziano** [comment]
Moving this to the new milestone because it is a low priority item to complete.
